security/clamav/src/opnsense/mvc/app/views/OPNsense/ClamAV/general.volt

```diff
@@ -34,6 +34,7 @@ POSSIBILITY OF SUCH DAMAGE.
 
 <ul class="nav nav-tabs" data-tabs="tabs" id="maintabs">
     <li class="active"><a data-toggle="tab" href="#general">{{ lang._('General') }}</a></li>
+    <li><a data-toggle="tab" href="#lists">{{ lang._('Lists') }}</a></li>
```

Comment:
I would call that external signatures

Reply:
Would be a bit long for a naming a tab 🤔

Reply:
That may be a question of taste. If you want it as is, it's your plugin :)

Reply:
I changed to `Signatures`